lib/sql.js

```diff
@@ -1207,15 +1207,25 @@ SQLConnector.prototype.buildOrderBy = function(model, order) {
   if (typeof order === 'string') {
     order = [order];
   }
+  const props = self.getModelDefinition(model).properties;
   const clauses = [];
   for (let i = 0, n = order.length; i < n; i++) {
     const t = order[i].split(/[\s,]+/);
+    const key = t[0];
+    if (!props[key]) {
+      // Unknown property, ignore it
+      debug('Unknown property %s is skipped for model %s', key, model);
```

Comment:
We shouldn't be ignoring unknown properties, see the discussions in https://github.com/strongloop/loopback-connector/issues/125 and https://github.com/strongloop/loopback-connector/pull/132.

Reply:
Ok makes sense, but the other issues don't state a solution - would you like me to throw an error instead?

Reply:
Yes please, let's throw an Error. See https://github.com/strongloop/loopback-connector/pull/132#discussion_r215983213.

Ideally, the error should have a `code` property set to a unique string that we can use to detect this particular error programmatically, and also `statusCode` set to `400` to ensure our REST API will respond with a status code indicating that the problem is at the client side.

Reply:
Aure I will revise to add status codes

Reply:
Done